lib/xcode/install.rb

```diff
@@ -105,9 +105,8 @@ def install_dmg(dmgPath, suffix = '', switch = true, clean = true)
       FileUtils.rm_f(dmgPath) if clean
     end
 
-    def install_version(version, switch = true, clean = true, install = true, progress = true)
-      return if version.nil?
-      dmg_path = get_dmg(version, progress)
+    def install_version(version, switch = true, clean = true, install = true, progress = true, url)
```

Comment:
it could, although the only place it's used in this codebase provides a value for it